src/Illuminate/Mail/Transport/SesTransport.php

```diff
@@ -57,7 +57,7 @@ public function stop()
     public function send(Swift_Mime_Message $message, &$failedRecipients = null)
     {
         return $this->ses->sendRawEmail([
-            'Source' => key($message->getSender()),
+            'Source' => key(is_null($message->getSender()) ? $message->getFrom() : $message->getSender()),
```

Comment:
This doesn't look right. Just do: `$message->getSender() ?: $message->getFrom()`